templates/guides/opting_into_beta_resources.md.tmpl

```diff
@@ -0,0 +1,44 @@
+---
+page_title: "Configuring Beta Resources in the STACKIT Terraform Provider"
+---
+# Configuring Beta Resources in the STACKIT Terraform Provider
+
+### Overview
+
+This guide explains how to opt into beta resources within the STACKIT Terraform provider. Beta resources are new services and features from the STACKIT cloud that are still in development and do not yet have a stable API. 
+
+Opting into beta functionality allows users to experiment with these cutting-edge features without compromising the stability of other resources and the provider itself.
+
+Enabling beta resources allows you to try out new features and services before they are officially released. While this can be exciting, it's important to remember that beta resources may not be as stable as their fully released counterparts. Use them with caution and provide feedback to help improve these services.
+
+### The Process of Opting into the Beta
+
+To use beta resources in the STACKIT Terraform provider, you have two options:
+
+#### Option 1: Provider Configuration
+
+Set the `enable_beta_resources` option in the provider configuration. This is a boolean attribute that can be either `true` or `false`.
+
+```hcl
+provider "stackit" {
+  region                = "eu01"
+  enable_beta_resources = true
+}
+```
+
+#### Option 2: Environment Variable
+
+Set the `STACKIT_TF_ENABLE_BETA_RESOURCES` environment variable to `"true"` or `"false"`. Other values will be ignored and will produce a warning.
+
+```sh
+export STACKIT_TF_ENABLE_BETA_RESOURCES=true
+```
+
+> **Note**: **The environment variable takes precedence over the provider configuration option.**
+>
+> This means that if the `STACKIT_TF_ENABLE_BETA_RESOURCES` environment variable is set to a valid value (`"true"` or `"false"`), it will override the `enable_beta_resources` option specified in the provider configuration.
+### Listing Beta Resources
+
```

Comment:
```suggestion
> This means that if the `STACKIT_TF_ENABLE_BETA_RESOURCES` environment variable is set to a valid value (`"true"` or `"false"`), it will override the `enable_beta_resources` option specified in the provider configuration.

### Listing Beta Resources

```

Reply:
I think a new line was missing